### PR TITLE
[CBRD-24477] Add basic test cases for PL/CSQL

### PR DESCRIPTION
**Pull request by hgryoo**
http://jira.cubrid.org/browse/CBRD-24477

### CODE REVIEW

File: sql/_35_fig_cake/plcsql/basic/answers/pingpong.answer
Comment (hyunikn) [inline sql/_35_fig_cake/plcsql/basic/answers/pingpong.answer:1]: Maybe, we need to remove this empty file.

File: sql/_35_fig_cake/plcsql/basic/cases/test_ddl.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
+create or replace procedure test_ddl() as
+    i int;
+    n varchar;
+    p varchar;
+
+    new_table VARCHAR := 'a_tbl1';
+begin
+    EXECUTE IMMEDIATE 'drop table if exists ' || new_table;
+    EXECUTE IMMEDIATE 'CREATE TABLE ' || new_table || ' (id INT UNIQUE, name VARCHAR, phone VARCHAR DEFAULT ''000-0000'');';
+    DBMS_OUTPUT.put_line ('creating ' || new_table || ' table is succeed!');
```
Comment (hyunikn) [inline sql/_35_fig_cake/plcsql/basic/cases/test_ddl.sql]: (minor) is succeed --> succeeded

File: sql/_35_fig_cake/plcsql/basic/cases/test_static_sql_dml.sql
Comment (hyunikn) [inline sql/_35_fig_cake/plcsql/basic/cases/test_static_sql_dml.sql:1]: Maybe, we need to drop table 'test_tbl' at last.

File: sql/_35_fig_cake/plcsql/basic/cases/test_static_sql_query.sql
Comment (hyunikn) [inline sql/_35_fig_cake/plcsql/basic/cases/test_static_sql_query.sql:1]: Maybe, we need to drop table 'history' at last.

File: sql/_35_fig_cake/plcsql/basic/cases/test_tcl.sql
Comment (hyunikn) [inline sql/_35_fig_cake/plcsql/basic/cases/test_tcl.sql:1]: Maybe, we need to drop tables test_tcl_tbl and test_tcl_tbl2

**Comment by hgryoo** [comment]
@hyunikn 
I've cleanup used objects at the end of each case. (512090b)
Thank you for the review.

**Review by hyunikn — APPROVED** [review]

**Comment by hgryoo** [comment]
@ssihil please review this PR after releasing the fig.

File: sql/_35_fig_cake/plcsql/basic/cases/test_serial.sql [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+begin
+    i := demo_pl_serial.NEXT_VALUE;
+    return i;
+end;
+
+CAll demo_global_semantics_serial ();
+CAll demo_global_semantics_serial ();
+CAll demo_global_semantics_serial ();
+CAll demo_global_semantics_serial ();
+
```
Comment (hiclass) [inline sql/_35_fig_cake/plcsql/basic/cases/test_serial.sql:15]: select demo_pl_serial.CURRVAL [from dual] ;
or
select current_val from db_serial where name = 'demo_pl_serial' ;
or
select demo_global_semantics_serial ();

would be need to check  created function works well.

**Pull request merged**